### ISSUE TITLE
sharing menu is empty

### ISSUE THREAD

**Issue by paskalito** [nextcloud/server#13753]
### Steps to reproduce
1. login to nextcloud 15.02
2. click on the share button of a folder you shared before

### Expected behaviour
i see with whom i shared the folder and can edit that

### Actual behaviour
noone gets loaded
(still i can add new users to the share)

tried firefox, deleted cache, turned off adblocker.
opera, standard config

### Server configuration

**Operating system**:
 debian 9
**Web server:**
apache
**Database:**
mysql
**PHP version:**
7.3
**Nextcloud version:** (see Nextcloud admin page)
15.02
**Updated from an older Nextcloud/ownCloud or fresh install:**
updatet from 15.00
**Where did you install Nextcloud from:**
i think originaly from website and updated since then
**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Accessibility	1.1.0	Offiziell	
Activity	2.8.2	Offiziell	
AppOrder	0.6.0		
Audio Player	2.5.1		
Auditing / Logging	1.5.0	Offiziell	
Brute-force settings	1.3.0	Offiziell	
Calendar	1.6.4	Offiziell	
Collaborative tags	1.5.0	Offiziell	
Comments	1.5.0	Offiziell	
Contacts	3.0.2	Offiziell	
Deck	0.5.2		
Deleted files	1.5.0	Offiziell	
External storage support	1.6.0	Offiziell	
Federation	1.5.0	Offiziell	
File sharing	1.7.0	Offiziell	
First run wizard	2.4.0	Offiziell	
Gallery	18.2.0	Offiziell	
Group folders	2.0.2	Offiziell	
Log Reader	2.0.0	Offiziell	
Mail	0.11.0	Offiziell	
Metadata	0.8.0		
Monitoring	1.5.0	Offiziell	
Nextcloud announcements	1.4.0	Offiziell	
Notes	2.5.1	Offiziell	
Notifications	2.3.0	Offiziell	
Password policy	1.5.0	Offiziell	
PDF viewer	1.4.0	Offiziell	
Polls	0.9.5		
Radio	0.6.4		
Retention	1.4.0	Offiziell	
Share by mail	1.5.0	Offiziell	
Social	0.1.3		
Social sharing via Diaspora	1.0.4	Offiziell	
Support	1.0.0	Offiziell	
Talk	5.0.0	Offiziell	
Tasks	0.9.8		
Text editor	2.7.0	Offiziell	
Theming	1.6.0	Offiziell	
TwoFactor via nextcloud notification	1.0.2		
Unsplash	1.1.3		
Update notification	1.5.0	Offiziell	
Usage survey	1.3.0	Offiziell	
Versions	1.8.0	Offiziell	
Video player	1.4.0	Offiziell	
Workflow external scripts	1.0.0		
Default encryption module	2.3.0	Offiziell	
LDAP user and group backend
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
CONFIG = array (
  'instanceid' => ' ',
  'passwordsalt' => ' ',
  'secret' => ' ',
  'trusted_domains' => 
  array (
    0 => ' ',
  ),
  'datadirectory' => '/',
  'overwrite.cli.url' => '',
  'htaccess.RewriteBase' => '/owncloud',
  'dbtype' => 'mysql',
  'version' => '15.0.2.0',
  'dbname' => ' ',
  'dbhost' => 'localhost',
  'dbtableprefix' => ' ',
  'dbuser' => ' ',
  'dbpassword' => '  ',
  'logtimezone' => 'UTC',
  'installed' => true,
  'appstore.experimental.enabled' => true,
  'mail_smtpmode' => ' ',
  'mail_smtpsecure' => ' ',
  'mail_from_address' => ' ',
  'mail_domain' => ' ',
  'mail_smtpauthtype' => ' ',
  'mail_smtpauth' =>  ,
  'mail_smtphost' => ' de',
```

</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
firefox, opera
**Operating system:**
Manjaro
#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

... this is 60mb with loads of private data in it. i share the two most recent/often occuring fatal log entrys

```
[webdav] Fatal: Sabre\DAV\Exception\UnsupportedMediaType: Validation error in iCalendar: More than one master object was found for the object with UID 6cfcef75-4302-42eb-b7b8-81d11e76fc2c at <<closure>>

 0. /var/www/html/owncloud/3rdparty/sabre/dav/lib/CalDAV/Plugin.php line 772
    validateICalendar("BEGIN:VCALENDAR ... n", "calendars/xxxx ... s", false, Sabre\HTTP\Reque ... "}, Sabre\HTTP\Response {}, false)
 1. <<closure>>
    beforeWriteContent("calendars/xxxx ... s", OCA\DAV\CalDAV\CalendarObject {}, "BEGIN:VCALENDAR ... n", false)
 2. /var/www/html/owncloud/3rdparty/sabre/event/lib/EventEmitterTrait.php line 105
    call_user_func_array([OCA\DAV\CalDAV\ ... "], ["calendars/pasc ... e])
 3. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 1128
    emit("beforeWriteContent", ["calendars/pasc ... e])
 4. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php line 513
    updateFile("calendars/xxxx ... s", "BEGIN:VCALENDAR ... n", null)
 5. <<closure>>
    httpPut(Sabre\HTTP\Reque ... "}, Sabre\HTTP\Response {})
 6. /var/www/html/owncloud/3rdparty/sabre/event/lib/EventEmitterTrait.php line 105
    call_user_func_array([Sabre\DAV\CorePlugin {},"httpPut"], [Sabre\HTTP\Requ ... }])
 7. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 479
    emit("method:PUT", [Sabre\HTTP\Requ ... }])
 8. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 254
    invokeMethod(Sabre\HTTP\Reque ... "}, Sabre\HTTP\Response {})
 9. /var/www/html/owncloud/apps/dav/lib/Server.php line 301
    exec()
10. /var/www/html/owncloud/apps/dav/appinfo/v2/remote.php line 35
    exec()
11. /var/www/html/owncloud/remote.php line 163
    require_once("/var/www/html/o ... p")

PUT /owncloud/remote.php/dav/calendars/xxx/0e9363c8-3985-421f-9669-ddd0cbd5de0c_shared_by_xxxx/6cfcef75-4302-42eb-b7b8-81d11e76fc2c.ics
from 62.102.148.68 by xxx at 2019-01-22T20:20:21+00:00

[remote] Error: TypeError: Return value of OCA\Files_Trashbin\Sabre\AbstractTrash::getSize() must be of the type int, float returned at <<closure>>

 0. /var/www/html/owncloud/apps/files_trashbin/lib/Sabre/PropfindPlugin.php line 78
    getSize()
 1. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/PropFind.php line 98
    OCA\Files_Trashbin\Sabre\{closure}("*** sensitive parameters replaced ***")
 2. /var/www/html/owncloud/apps/files_trashbin/lib/Sabre/PropfindPlugin.php line 79
    handle("{http://owncloud.org/ns}size", Closure {})
 3. <<closure>>
    propFind(Sabre\DAV\PropFind {}, OCA\Files_Trashbin\Sabre\TrashFolder {})
 4. /var/www/html/owncloud/3rdparty/sabre/event/lib/EventEmitterTrait.php line 105
    call_user_func_array([OCA\Files_Trash ... "], [Sabre\DAV\PropF ... }])
 5. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 1059
    emit("propFind", [Sabre\DAV\PropF ... }])
 6. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 981
    getPropertiesByNode(Sabre\DAV\PropFind {}, OCA\Files_Trashbin\Sabre\TrashFolder {})
 7. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 1666
    getPropertiesIteratorForPath("trashbin/xxxx/trash", ["{http://nextcl ... "], 1)
 8. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php line 355
    generateMultiStatus(Generator {}, false)
 9. <<closure>>
    httpPropFind(Sabre\HTTP\Reque ... "}, Sabre\HTTP\Response {})
10. /var/www/html/owncloud/3rdparty/sabre/event/lib/EventEmitterTrait.php line 105
    call_user_func_array([Sabre\DAV\CorePlugin {},"httpPropFind"], [Sabre\HTTP\Requ ... }])
11. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 479
    emit("method:PROPFIND", [Sabre\HTTP\Requ ... }])
12. /var/www/html/owncloud/3rdparty/sabre/dav/lib/DAV/Server.php line 254
    invokeMethod(Sabre\HTTP\Reque ... "}, Sabre\HTTP\Response {})
13. /var/www/html/owncloud/apps/dav/lib/Server.php line 301
    exec()
14. /var/www/html/owncloud/apps/dav/appinfo/v2/remote.php line 35
    exec()
15. /var/www/html/owncloud/remote.php line 163
    require_once("/var/www/html/o ... p")

PROPFIND /owncloud/remote.php/dav/trashbin/xxxx/trash
from 87.123.193.166 by xxxx at 2019-01-22T17:26:49+00:00

**Comment by dburianov** [comment]
> I had the same issue in a new installation of Nextcloud 15.0.2, without upgrading, but with plugins. After making a fresh one and enabling each plugin or new feature one by one, I can confirm this issue is caused by the **official Social Share plugins** (in my case, email one, but it seems to happen also with Twitter and probably all of them).
> 
> After disabling them, my share panel seems to work fine even with LDAP, External storage, S3 primary storage, etc.

Thanks, I have a similar bug

**Comment by juliushaertl** [comment]
Fixed with #13738

**Comment by juliushaertl** [comment]
@paskalito The fix is included in the server code and will be shipped with the next maintenance release of 15. :wink:
